bundles/statistics/statsgrid2016/plugin/ClassificationToolPlugin.js

```diff
@@ -166,6 +170,35 @@ Oskari.clazz.define('Oskari.statistics.statsgrid.plugin.ClassificationToolPlugin
         },
         stopPlugin: function(){
             this.teardownUI(true);
+        },
+        _createEventHandlers: function () {
+            return {
+                /**
+                 * @method AfterMapMoveEvent
+                 * Shows map center coordinates after map move
+                 */
+                AfterMapMoveEvent: function () {
```

Comment:
Why does moving the map trigger size check? Shouldn't this be something like MapSizeChangedEvent?